src/cc_catalog_airflow/dags/provider_api_scripts/wikimedia_commons.py

```diff
@@ -323,6 +340,7 @@ def _create_meta_data_dict(image_data):
     meta_data = {}
     global_usage_length = len(image_data.get('globalusage', []))
     image_info = _get_image_info_dict(image_data)
+    date_originally_created, last_modified_at_source = _extract_date_info(image_info)
```

Comment:
Unfortunately, the new variables push this line over the allowed length of 79 characters.

Reply:
So sorry about this! I fixed the formatting issues in the main code and the test file. I think it's ok to use the default code styling that comes with pycharm because it's supposed to be the same as PEP8 (https://blog.jetbrains.com/pycharm/2013/02/long-awaited-pep-8-checks-on-the-fly-improved-doctest-support-and-more-in-pycharm-2-7/)? However I had to manually set the maximum allowed length to 79 because it was 120 by default.